### PR TITLE
[irods/irods_3062] better enumerate the operators for imeta qu

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

File: src/imeta.cpp [hunk 1]
```diff
@@ -2452,12 +2452,15 @@ usage( const char *subOpt ) {
         }
         if ( strcmp( subOpt, "qu" ) == 0 ) {
             char *msgs[] = {
-                " qu -d|C|R|u AttName Op AttVal [...] (Query objects with matching AVUs)",
+                " qu -d|C|R|u AttName Operator AttVal [...] (Query objects with matching AVUs)",
                 "Query across AVUs for the specified type of item",
                 "Example: qu -d distance '<=' 12",
                 " ",
+                "The Operator can be any of the following:",
+                " <, >, <=, >= <>, =, !=, like, not like",
```
Comment (alanking) [inline src/imeta.cpp:2460]: Should there be a comma between >= and <>?

Comment (trel) [reply src/imeta.cpp]: done

**Review by alanking — APPROVED** [review]

**Comment by trel** [comment]
#'d

**Comment by alanking** [comment]
Should this be cherry-picked to 4-2-stable?

**Comment by trel** [comment]
indeed, done.

**Pull request merged**